client-ng1/src/app/js/admin/cfp/cfp.controller.js

```diff
@@ -22,8 +22,42 @@
           ctrl.proposals = response.data;
         }
       );
+      $http.get('app/cfp/proposal/votes').then(
+          function(response){
+            ctrl.votesMappedByProposalId = {};
+            angular.forEach(response.data, function(data) {
+              ctrl.votesMappedByProposalId[data.proposal.id] = data.voteValue;
```

Comment:
For the full data it's not ilmportant in admin screens
The big problem with Angular is when you attach a long collection in scope with double data binding. If you use pagination the performances will be acceptables